src/components/AssetManagementPanel/AssetManagementForm/FormVariants/NominateBeneficiaryHolder/NominateBeneficiaryHolder.tsx

```diff
@@ -53,9 +52,7 @@ export const NominateBeneficiaryHolderForm = ({
   }, [isConfirmed, newBeneficiary, showOverlay, setFormActionNone]);
 
   const isValidNomination = () => {
```

Comment:
how about:

```suggestion
const isInvalidNomination = !newBeneficiary || newBeneficiary === beneficiary || !isAddress(newBeneficiary);
```

then at the button there:

```
disabled={isInvalidNomination || isPendingConfirmation}
```

Reply:
ok